src/model/selection.js

```diff
@@ -7,14 +7,14 @@
  * @module engine/model/selection
  */
 
-import Position from './position';
+import CKEditorError from '@ckeditor/ckeditor5-utils/src/ckeditorerror';
```

Comment:
Why did you changed the imports? Please revert this change unless it's a must-have in this ticket.